### PR TITLE
Fix formatting issues in file.

### PR DESCRIPTION
**Pull request by mfilippov**
I removed double spaces and double semi-colon in the ContolPaint.cs.

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/ControlPaint.cs [hunk 1]
```diff
@@ -2495,16 +2489,16 @@ public static void DrawMenuGlyph(Graphics graphics, int x, int y, int width, int
             matrix.Matrix02 = -normBlackBlue;
             matrix.Matrix03 = -normBlackAlpha;
 
-            matrix.Matrix10 =  normWhiteRed;
```
Comment (zsd4yr) [inline src/System.Windows.Forms/src/System/Windows/Forms/ControlPaint.cs]: I think we may wish to keep these in order to keep the names in column regardless of the - symbol

**Comment by zsd4yr** [comment]
One place where I think the extra spaces may have been intentional. Otherwise LGTM

**Comment by mfilippov** [comment]
@zsd4yr fixed.

**Comment by zsd4yr** [comment]
Thanks @mfilippov --this is a nice cleanup :shipit:

**Pull request merged**